plugins/module_utils/mso.py

```diff
@@ -853,6 +859,13 @@ def lookup_roles(self, roles, ignore_not_found_error=False):
             ids.append(dict(roleId=r.get("id"), accessType=access_type))
         return ids
 
+    def lookup_site_type(self, s):
```

Comment:
Can you rename "s" to a more descriptive value. Found it confusing to understand where this platform key is now coming from. This is also due to the fact that in the module class we also have a platform defined with values "local", "nd" and "mso".